istioctl/cmd/proxystatus.go

```diff
@@ -104,18 +157,39 @@ Retrieves last sent and last acknowledged xDS sync from Istiod to each Envoy in
 			if err != nil {
 				return err
 			}
-			sw := pilot.StatusWriter{Writer: c.OutOrStdout()}
+			sw := pilot.StatusWriter{Writer: c.OutOrStdout(), XDSCols: commonXDSCols}
+			sw.XDSCols = getColumnHeaders(xdsStatusCols)
 			return sw.PrintAll(statuses)
 		},
 	}
 
 	opts.AttachControlPlaneFlags(statusCmd)
+	statusCmd.PersistentFlags().BoolVarP(&showAllXDSStatuses, "all-xds-statuses", "A", false,
```

Comment:
Can the flag be something shorter, like `--all` or `--wide`?  Like how Kubernetes users are used to `kubectl get pods -o wide`?

Reply:
I will go with `--wide`. Like in Kubernetes, it means more columns.

Reply:
Done.